backend/db/migrations/20200710134411-changeEventTypeToEnum.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { PrintfulWebhookEvents, ExternalServices } = require('../../utils/enums')
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
ah cool. appreciate you updating this to an enum. will make our schema cleaner :)